collectors/python.d.plugin/postgres/README.md

```diff
@@ -68,6 +68,11 @@ Following charts are drawn:
 
     -   locks
 
+12. **Standby lag**
+
+    - WAL offset
+    - replication lag in seconds
+
```

Comment:
You didn't add the chart _Average number of blocking transactions_.

Dimensions for _Standby lag_ are incorrect.

Reply:
ok, now I see those names come directly from chart definitions